### PR TITLE
Feature/fixes

### PR DESCRIPTION
**Pull request by boazpoolman**
### What does it do?

1. It now queries all languages in the `/all` endpoint
2. The santize function exported from `@strapi/utils` has been renamed. The PR reflects that change.

### Additional info

I've allready released these two things. Though I've based that of the previous releases. So all the Typescript stuff has not yet been released. This is just a PR to get these fixes back into master as well so that they won't be overwritten when we do make a release of master.

### CODE REVIEW

File: server/content-api/services/by-path.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ export default () => ({
           $notNull: true,
         } : {},
       },
+      locale: 'all',
```
Comment (Alexnortung) [inline server/content-api/services/by-path.ts:37]: Maybe this has something to do with something else, but why is this necessary? Isn't Strapi always using all locales?

Comment (Alexnortung) [reply server/content-api/services/by-path.ts:37]: Oh nvm, I just checked, it is not by default sorry.

**Review by Alexnortung — APPROVED** [review]

File: server/content-api/services/by-path.ts (hunk 1 above)
Comment (boazpoolman) [reply server/content-api/services/by-path.ts:37]: Don't worry. I'ts a good point of discussion.
I made the assumption that it is best to query all pages of all languages in the `/all` endpoint.
Do you agree?

Comment (Alexnortung) [reply server/content-api/services/by-path.ts:37]: I defenitely agree and I also agree that it makes sense to query all locales when looking for a specific path :)
